<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(22, 26, 48);
background-color: rgb(244, 244, 244);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_navbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(20, 20, 20);
border: 2px solid rgb(22, 26, 48); </string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(39, 39, 39);</string>
    </property>
    <property name="title">
     <string>Выберите магазин</string>
    </property>
    <widget class="QComboBox" name="comboBox__shop_select">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(39, 39, 39);
background-color: rgb(179, 179, 179);
background-color: rgb(225, 225, 225);
border:1px  solid rgb(39, 39, 39);
border-radius:10px;
background-color: rgb(182, 187, 196);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px  solid rgb(39, 39, 39);

border-radius:10px;
background-color: rgb(161, 161, 161);

</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px  solid rgb(39, 39, 39);
border-radius:10px;

background-color: rgb(161, 161, 161);</string>
    </property>
    <property name="text">
     <string>Посмотреть логи</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcome">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 236, 229);


border: 0px;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Добро Пожаловать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px  solid rgb(39, 39, 39);

border-radius:10px;
background-color: rgb(161, 161, 161);

</string>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <zorder>label_welcome</zorder>
   <zorder>groupBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_quit</zorder>
  </widget>
  <widget class="QGroupBox" name="content">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>1141</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 48, 77);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_registration">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px  solid rgb(39, 39, 39);
border-radius:0px;
background-color: rgb(245, 245, 245);</string>
    </property>
    <property name="title">
     <string>Регистрация</string>
    </property>
    <widget class="QLabel" name="label_enterFIO">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px</string>
     </property>
     <property name="text">
      <string>Введите ФИО</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_registration">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_registrate">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Создать билет</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_logs">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>1011</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px  solid rgb(39, 39, 39);

border-radius:2px;
background-color: rgb(245, 245, 245);</string>
    </property>
    <property name="title">
     <string>логи</string>
    </property>
    <widget class="QLabel" name="label_logs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>981</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_show_films">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Посмотреть все фильмы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_show_films_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Посмотреть всеx юзеров</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_show_films_3">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Посмотреть все аренды</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_ArendaTake">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>311</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px  solid rgb(39, 39, 39);

border-radius:10px;
background-color: rgb(245, 245, 245);</string>
    </property>
    <property name="title">
     <string>Взять в прокат</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="text">
      <string>Итого</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_check_cost">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Посмотреть стоимость</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_take">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Взять</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="text">
      <string>Выберите фильм</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cost">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_film">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:2px;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_period">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:2px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_ArendaReturn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px  solid rgb(39, 39, 39);
border-radius:10px;
background-color: rgb(245, 245, 245);
</string>
    </property>
    <property name="title">
     <string>Вернуть </string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="text">
      <string>Выберите фильм</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_return">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вернуть</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_film_return">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:2px;</string>
     </property>
    </widget>
    <zorder>label_6</zorder>
    <zorder>pushButton_return</zorder>
    <zorder>comboBox_film_return</zorder>
    <zorder>groupBox_logs</zorder>
   </widget>
  </widget>
  <zorder>content</zorder>
  <zorder>groupBox_navbar</zorder>
  <zorder>label_logs</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
